notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        # There are roughly 2x as many moves as rows,
</pre><p>Why "roughly 2x"? 1.48 to 2.22 is quite a range away from 2x. Sometimes the number of moves in a game is odd, but that would only explain a tiny fraction of this range.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Good point. I don't think my explanation here is a very good one. I'll ask this on Slack and see if others have ideas.